ra2ce/network/network_config_data/network_config_data.py

```diff
@@ -82,6 +82,9 @@ class HazardSection:
     aggregate_wl: AggregateWlEnum = field(default_factory=lambda: AggregateWlEnum.NONE)
     hazard_crs: str = ""
     scenario_cost: list[float] = field(default_factory=list)
+    # Should a hazard overlay be created for the segmented network.
```

Comment:
Make it more directive `When true the overlay network is segmented`.

Reply:
Done